welcome.md

```diff
@@ -0,0 +1,25 @@
+# Welcome to Ruby Tuesday!
+Please grab some food and make yourself comfortable.
+
+## Getting Connected
+* Slack: wwc-atx.slack.com
+* Twitter: @wwcodeatx
+* Web: womenwhocode.com/austin: Coding Resources, Job Board, Meetups, and MOAR!
+
+## Your Organizers
+* Miriam Hayes: contact info
+* Anne Richardson:  contact info
```

Comment:
We should put some kind of contact info here. I'm not sure how to do this anymore! Generally i prefer for people to DM me via slack, but if they're not members, how do they do that?

Also, do we have an easy way to add people to the slack group?

Reply:
People sometimes post in the Meetup.

`invite_people name@internet.com`

Reply:
Is `invite_people name@internet.com` a slack bot integration? How do we use it?

Reply:
Yes. Slackbot. 
`/invite_people miriam@internet.org`